versions.yaml

```diff
@@ -221,14 +221,23 @@ externals:
   containerd:
     description: |
       Containerd for Kubernetes Container Runtime Interface.
-    # CCv0 is using our fork of containerd as the changes can't be merged yet
-    # DO NOT TOUCH DURING MERGES FROM MAIN TO CCv0
-    url: "github.com/confidential-containers/containerd"
-    # yamllint disable-line rule:line-length
-    tarball_url: "https://github.com/confidential-containers/containerd/releases/download"
-    version: "v1.6.8.2"
-    # CCv0 needs to know about the specific branch for the integration tests
-    branch: "CC-main"
+    forked:
+      # CCv0 is using our fork of containerd as the changes can't be merged yet
+      # DO NOT TOUCH DURING MERGES FROM MAIN TO CCv0
+      url: "github.com/confidential-containers/containerd"
+      # yamllint disable-line rule:line-length
+      tarball_url: "https://github.com/confidential-containers/containerd/releases/download"
+      version: "v1.6.8.2"
+      # CCv0 needs to know about the specific branch for the integration tests
+      branch: "CC-main"
+    upstream:
+      # DO NOT TOUCH DURING MERGES FROM MAIN TO CCv0
+      url: "github.com/containerd/containerd"
+      # yamllint disable-line rule:line-length
+      tarball_url: "https://github.com/containerd/containerd/releases/download"
+      version: "v1.7.2"
```

Comment:
Are you happy for us to be ahead of the `main` versions.yaml here? I'm not against this, but it means that we'd probably need to bump containerd in `main` before we merge this function to `main` if we aren't confident that it works on 1.6.8

Reply:
So, we're already testing on `main` with the `latest` containerd (at least the cri-containerd tests).  The k8s tests are relying on what k3s is using, which is also 1.7+

Reply:
It's very hard to compare this with `main`, as we have changed the way we consume those for the tests on main.

Reply:
Cool - fair point, if AKS is using containerd 1.7+ and we can kill on Jenkins soon then this problem goes away a bit